packages/swapper/src/types.ts

```diff
@@ -1,16 +1,49 @@
 import type { StdSignDoc } from '@keplr-wallet/types'
 import type { AssetId, ChainId, Nominal } from '@shapeshiftoss/caip'
-import type { CosmosSdkChainId, EvmChainId, UtxoChainId } from '@shapeshiftoss/chain-adapters'
+import type {
+  ChainAdapter,
+  CosmosSdkChainAdapter,
+  CosmosSdkChainId,
+  EvmChainAdapter,
+  EvmChainId,
+  UtxoChainAdapter,
+  UtxoChainId,
+} from '@shapeshiftoss/chain-adapters'
 import type { BTCSignTx, HDWallet } from '@shapeshiftoss/hdwallet-core'
 import type {
   AccountMetadata,
   Asset,
   AssetsByIdPartial,
+  KnownChainIds,
   PartialRecord,
   UtxoAccountType,
 } from '@shapeshiftoss/types'
 import type { evm, TxStatus } from '@shapeshiftoss/unchained-client'
 import type { Result } from '@sniptt/monads'
+import type { ethers as ethersV5 } from 'ethers5'
+import type { Chain, PublicClient, Transport } from 'viem'
+
+import type { makeSwapperAxiosServiceMonadic } from './utils'
+
+// TODO: Rename all properties in this type to be camel case and not react specific
```

Comment:
an example of something I want to clean up after this is merged

Reply:
Agree it sucks as it's not react specific anymore, but this is just about naming anyway